tests/CMakeLists.txt

```
@@ -0,0 +1,186 @@
+include(AddFdbTest)
+
+# We need some variables to configure the test setup
+set(ENABLE_BUGGIFY OFF CACHE BOOL "Enable buggify for tests")
```

```suggestion
set(ENABLE_BUGGIFY ON CACHE BOOL "Enable buggify for tests")
```

I'd think that for most testing situations, you'd probably want buggify to be on in hopes of triggering more bugs?